README.md

```diff
@@ -122,13 +145,20 @@ e.g.
     "title_number": "DN100",
     "borrowers": [
         {
-            "forename": "fred",
-            "middle_name": "joe",
-            "surname": "bloggs"
+            "forename": "lisa",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
+            "phone_number": "07502154062"
         },
         {
-            "forename": "lisa",
-            "surname": "bloggette"
+            "forename": "frank",
+            "middle_name": "ann",
+            "surname": "bloggette",
+            "address": "Example address with postcode, PL14 3JR",
+            "dob": "23/01/1986",
```

Comment:
Michael, its like we're the same person. I had that conversation on slack earlier and its been added to the backlog for later as we need sign off.